packages/odyssey-react-mui/src/theme/components.tsx

```diff
@@ -303,18 +303,34 @@ export const components = ({
     },
     MuiAlertTitle: {
       styleOverrides: {
-        root: {
+        root: ({ ownerState }) => ({
           marginBlockStart: 0,
           marginBlockEnd: odysseyTokens.Spacing2,
           lineHeight: odysseyTokens.TypographyLineHeightHeading6,
           fontSize: odysseyTokens.TypographySizeHeading6,
           fontWeight: odysseyTokens.TypographyWeightHeading,
           fontFamily: odysseyTokens.TypographyFamilyHeading,
 
+          ...(ownerState.severity === "info" && {
+            color: odysseyTokens.PalettePrimaryDark,
+          }),
+
+          ...(ownerState.severity === "error" && {
+            color: odysseyTokens.PaletteDangerDarker,
+          }),
+
+          ...(ownerState.severity === "success" && {
+            color: odysseyTokens.PaletteSuccessDarker,
+          }),
+
+          ...(ownerState.severity === "warning" && {
+            color: odysseyTokens.PaletteWarningDarker,
+          }),
```

Comment:
We have an area in the code where we're not using `ownerState` and using classNames instead. Can you add a ticket to fix those?